process/block/metablock.go

```diff
@@ -1247,7 +1247,7 @@ func (mp *metaProcessor) CommitBlock(
 	lastMetaBlockHash := mp.blockChain.GetCurrentBlockHeaderHash()
 	mp.updateState(lastMetaBlock, lastMetaBlockHash)
 
-	err = mp.blockChain.SetCurrentBlockHeader(header)
+	err = mp.blockChain.SetCurrentBlockHeaderAndRootHash(header, header.RootHash)
```

Comment:
maybe get here as well the committed roothash? 
to cover also the case where we add scheduled System SC calls on metachain

Reply:
done